streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -86,7 +87,7 @@
     private boolean rebalanceInProgress = false;  // if we are in the middle of a rebalance, it is not safe to commit
 
     // includes assigned & initialized tasks and unassigned tasks we locked temporarily during rebalance
-    private Set<TaskId> lockedTaskDirectories = new HashSet<>();
+    private final Set<TaskId> lockedTaskDirectories = new HashSet<>();
```

Comment:
just noticed this.

Reply:
Has checkstyle just been dropping the ball lately? Could we be unknowingly suppressing this...?

Reply:
Yeah, I thought that was weird when I added the "final" checkstyle rule. Oh well...